src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -301,6 +313,8 @@ class ReviewCollapsibleChapter extends React.Component {
           data={props.form.data}
           updatePage={() => this.handleEdit(page.pageKey, false, page.index)}
           pagePerItemIndex={page.index}
+          schema={pageSchema}
+          uiSchema={pageUiSchema}
```

Comment:
Can you explain the significance of this change? I don't have much experience working on this code, so additional guidance would help me give a better and more efficient review.

Reply:
Since we're working in the `ReviewCollapsibleChapter` component, that means it's only rendered on the review & submit page. When you edit a page, the render switches from a `CustomPageReview` component to the `CustomPage` component which is the same component rendered within the form flow.

The `CustomPage` doesn't include any validation. If you add a `ui:validations` option, the check is only preformed on the review & submit page after submitting the form. To get around this, we need to manually check data validation within the `CustomPage`. And since we're already using `ui:validations`, we need to access the `uiSchema`.

When you compare the render of `CustomPage` in [ReviewCollapsibleChapter](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx) and [FormPage](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/forms-system/src/js/containers/FormPage.jsx#L185-L204), you'll see this diff:

```diff
 <page.CustomPage
+  key={page.pageKey} // rendering multiple pages
-  callOnContinue={callOnContinue} // replaced by `updatePage`
-  contentAfterButtons={contentAfterButtons} // save-in-progress link
-  contentBeforeButtons={contentBeforeButtons} // save-in-progress status
   data={props.form.data}
   goBack={noop} // noop because no used & prevents JS error
   goForward={noop} // noop because no used & prevents JS error
   goToPath={this.goToPath}
   name={page.pageKey}
-  onChange={this.onChange} // replaced by `updatePage`
   onReviewPage
-  onSubmit={this.onSubmit} // replaced by `updatePage`
   pagePerItemIndex={page.index}
-  schema={schema} // MISSING!
   setFormData={props.setData}
   title={page.title}
   trackingPrefix={props.form.trackingPrefix}
-  uiSchema={uiSchema} // MISSING!
+  updatePage={() => this.handleEdit(page.pageKey, false, page.index)}
   uploadFile={props.uploadFile}
 />
```

In this case, we need the `uiSchema` to check `ui:required` and `ui:validations` to ensure we're validating the page changes. `schema` was added to be thorough. This shouldn't require a document update (I still need to check) because the `CustomPage` should require it.